packages/openactive-integration-tests/test/features/leasing/named-leasing/implemented/test-opportunity-bookable-test.js

```diff
@@ -0,0 +1,67 @@
+const { expect } = require('chai');
+const moment = require('moment');
+const { FeatureHelper } = require('../../../../helpers/feature-helper');
+const { GetMatch, C1, C2 } = require('../../../../shared-behaviours');
+
+/**
+ * @typedef {import('chakram').ChakramResponse} ChakramResponse
+ */
+
+/**
+ * @param {() => ChakramResponse} getChakramResponse This is wrapped in a
+ *   function because the actual response won't be available until the
+ *   asynchronous before() block has completed.
+ */
+function itShouldReturnLeaseWithFutureExpiryDate(getChakramResponse) {
+  it('should return lease with future expiry date', () => {
+    const chakramResponse = getChakramResponse();
+    const leaseExpires = moment(chakramResponse.body.lease.leaseExpires);
+    expect(leaseExpires.isAfter()).to.equal(true);
+  });
+}
+
+FeatureHelper.describeFeature(module, {
+  testCategory: 'leasing',
+  testFeature: 'named-leasing',
+  testFeatureImplemented: true,
+  testIdentifier: 'test-opportunity-bookable',
```

Comment:
Also this identifier, and the filename, should be named with something more specific to the test itself:

```suggestion
  testIdentifier: 'lease-response-test',
```